.travis.yml

```diff
@@ -6,37 +6,39 @@ env:
     - CS=7.2
     - DBNAME=glpitest
     - OLDDBNAME=glpiupgradetest
-    - AFTER_SUCCESS_BRANCH=9.3/bugfixes
+    - AFTER_SUCCESS_BRANCH=9.4/bugfixes
   matrix:
     - GLPI_BRANCH=9.3.0
     - GLPI_BRANCH=9.3/bugfixes
-    - GLPI_BRANCH=master
+    - GLPI_BRANCH=9.4/bugfixes
+#    - GLPI_BRANCH=master
 
 php:
   - 5.6
   - 7.0
   - 7.1
   - 7.2
+  - 7.3
   - nightly
 
 matrix:
   allow_failures:
     - php: nightly
-    - env: GLPI_BRANCH=master
+#    - env: GLPI_BRANCH=master
 
 before_script:
   - "./tests/before_script.sh"
 
 script:
   - if [ "${TRAVIS_PHP_VERSION:0:3}" = "$CS" ] && [ "$GLPI_BRANCH" = "$AFTER_SUCCESS_BRANCH" ]; then COVERAGE="--nccfc CommonTreeDropdown CommonDropdown CommonDBTM CommonGLPI"; else COVERAGE="-ncc"; fi
   - if [ -e ../../scripts/cliinstall.php ] ; then php ../../scripts/cliinstall.php --db=$OLDDBNAME --user=root --tests ; fi
-  - if [ -e ../../tools/cliinstall.php ] ; then php  ../../tools/cliinstall.php --db=$OLDDBNAME --user=root --tests ; fi
+  - if [ -e ../../bin/console ] ; then php ../../bin/console glpi:database:install --db-name=$OLDDBNAME --db-user=root --config-dir=../../tests --no-interaction --force --no-plugins; fi
```

Comment:
`--no-plugins` should be removed now (see https://github.com/glpi-project/glpi/pull/5105)`